plugins/reporters/cyclonedx/src/main/kotlin/CycloneDxReporter.kt

```diff
@@ -390,7 +391,7 @@ class CycloneDxReporter : Reporter {
                     BomGeneratorFactory.createJson(schemaVersion, bomWithoutExtensibleTypes) as Any
                 }
 
-                else -> throw IllegalArgumentException("Unsupported CycloneDX file format '$fileFormat'.")
+                else -> throw IllegalArgumentException("Unsupported CycloneDX file extension '$fileExtension'.")
```

Comment:
What's the reason you choose to change the wording from "extension" to "format" ?

Reply:
Because the code now really works directly with file extensions compared to semantic names for file formats (via the enum) compared to before.